requirements.txt

```diff
@@ -15,3 +15,4 @@ rpmfile>=1.0.6
 zstandard; python_version >= "3.4"
 reportlab
 distro
+requests
```

Comment:
I've used requests to fetch the JSON data
I read through cvedb.py and there we use something called session to fetch the data. But I didn't get how to we used it in https://github.com/intel/cve-bin-tool/blob/444927906ae1ff707d7680a5d7eb10b57d342c1f/cve_bin_tool/cvedb.py#L119-L131 

can this be re-used over here?

Reply:
Used `urllib` instead :D